registry/registry.go

```diff
@@ -96,13 +69,13 @@ func (reg *registry) tagsToRepository(client Client, id image.Name, tags []strin
 	toFetch := make(chan string, len(tags))
 	fetched := make(chan result, len(tags))
 
-	for i := 0; i < reg.connections; i++ {
+	for i := 0; i < 100; i++ {
```

Comment:
Removed already.

Reply:
The field value is arbitrary itself (and replicated in 100 places), and I removed it. This was a temporary situation, since this was all replaced with just fetching the whole value.